packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -332,6 +354,20 @@ export default class extends Component {
                     {host.multipathing && <MultipathableSrs hostId={host.id} />}
                   </td>
                 </tr>
+                {this.state.schedGran !== 'MESSAGE_METHOD_UNKNOWN' && (
+                  <tr>
+                    <th>{_('hostSchedulerGranularity')}</th>
```

Comment:
As the issue suggests, we also need to mention somewhere that the host needs to be restarted after a change.